### PR TITLE
update release notes

### PR DESCRIPTION
**Pull request by hobadams**

### CODE REVIEW

File: docs/platform/release/2-7-13.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+N/A
+
+## New Features 🚀
+
+N/A
+
+## Bug Fixes 🐛
+
+- **Security Patch** this patch is recommended to be installed by all users. (`@deity/falcon-bigcommerce-api`)
+- Change the way the pending payment page redirects to be more efficient (demo v1 and demo v2)
```
Comment (Borales) [inline docs/platform/release/2-7-13.md:21]:
```suggestion
- Changed the way the pending payment page redirects to be more efficient (demo v1 and demo v2)
```

File: docs/platform/release/2-7-8.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+N/A
+
+## New Features 🚀
+
+N/A
+
+## Bug Fixes 🐛
+
+- Fix url resolver so it doesn't pass query string parameters to Magento (`@deity/falcon-magento2-api`)
```
Comment (Borales) [inline docs/platform/release/2-7-8.md:20]:
```suggestion
- Fixed url resolver so it doesn't pass query string parameters to Magento (`@deity/falcon-magento2-api`)
```

File: docs/platform/release/2-7-7.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+###### Release Date (2021-04-28)
+
+## Breaking Changes 💥
+
+N/A
+
+## New Features 🚀
+
+- Added the api package `@deity/falcon-file-redirect-api` to the Falcon server config for both demos
+- Upgrade `@deity/falcon-payments` and corresponding packages to `2.7.1` in both demos.
```
Comment (Borales) [inline docs/platform/release/2-7-7.md:17]:
```suggestion
- Upgraded `@deity/falcon-payments` and corresponding packages to `2.7.1` in both demos.
```

@hobadams could you please check all other lines? :)

**Review by Borales — APPROVED** [review]
@hobadams in overall - approved

but we need to decide whether we keep things "updated" or "update"

**Comment by hobadams** [comment]
> @hobadams in overall - approved
> 
> but we need to decide whether we keep things "updated" or "update"

I've changed them to be past tense, good point

**Pull request merged**